### ISSUE TITLE
The "render-expired-frames" command wont work

### ISSUE THREAD

**Issue by TheAstrio** [Genymobile/scrcpy#3649]
**Environment**
 - OS: Windows 11
 - scrcpy version: v1.25
 - installation method: Windows release
 - device model:  Xiaomi Redmi Note 10 Lite
 - Android version: 12

The Bug : 
After running scrcpy as Administrator on my laptop, I used the following command below
scrcpy --render-expired-frames

When running the above command, it gives me an error that the option was ignored. The output error is below

```
WARN : Option --render-expired-frames has been removed. This flag has been ignored.
```

**Comment by rom1v** [comment]
Yes, it was provided to configure an useless internal detail and has been removed. Why do you need it?

**Comment by TheAstrio** [comment]
> Yes, it was provided to configure an useless internal detail and has been removed. Why do you need it?

The feature actually used to somewhat work better when enabled(50-55fps). The frame output from software drops a lot of frames (I had a counter on both my phone and pc and they don't match as my game runs at 56-60fps but the software shows it in 25-40fps)
Is there gonna be a newer command added which will show the true frames 😢 
My phone is at an average of 2400x1080p on default and everything else works fine. The command I usually use is below
```
scrcpy --render-expired-frames --bit-rate=16M --max-fps=100
```

**Comment by rom1v** [comment]
> I had a counter on both my phone and pc and they don't match as my game runs at 56-60fps but the software shows it in 25-40fps

Enabling the option just changes the count, but not all "rendered" frame will actually be displayed (several OpenGL calls will occur within a single display "slot", since vsync is disabled), so it does not change anything in practice (except it may add delay).

See 55806e7d312f2acabd99d1a3c707f7e42fbf90e2.

Btw, you can remove `--max-fps=100` to mean "unlimited".

**Comment by TheAstrio** [comment]
> > I had a counter on both my phone and pc and they don't match as my game runs at 56-60fps but the software shows it in 25-40fps
> 
> Enabling the option just changes the count, but not all "rendered" frame will actually be displayed (several OpenGL calls will occur within a single display "slot", since vsync is disabled), so it does not change anything in practice (except it may add delay).
> 
> See [55806e7](https://github.com/Genymobile/scrcpy/commit/55806e7d312f2acabd99d1a3c707f7e42fbf90e2).
> 
> Btw, you can remove `--max-fps=100` to mean "unlimited".

Oh thanks, could you also show how I can enable vsync for this software? Or is it overall for the device
